docs/versioning.rst

```diff
@@ -7,10 +7,15 @@ The functionality is optional and has to be explicitly enabled, note that:
 
 * SCM tags are used for versioning
 
-* only git is supported currently (contributions for other SCMs welcome and should be easy, see `Scm class`_)
+# XXX: s/git/Git (but also throughout then) ?
+* only git is supported currently (contributions for other SCMs are welcome and should be easy, see `Scm class`_)
 
 * It can be a drop-in replacement for setuptools_scm_
 
+# XXX: also "dev" and "devcommit"?!
+# XXX: the code also handles "changes" (https://github.com/zsimic/setupmeta/blob/586102493f7167d47d633ff46eb9945ae36b730c/setupmeta/versioning.py#L305)
```

Comment:
`devcommit` is described at bottom of page (very similar to `dev`, so I didn't put a section for it, maybe we should :) )
`changes` was the old name for `distance`, wasn't sure anyone ever used it, so left it there just in case.